### PR TITLE
Change cipher links to coinbase wallet

### LINKED ISSUE #23: Update Cipher Link
Cipher browser was acquired by Coinbase

Cipher deeplink should be replaced with coinbase wallet
Coinbase Wallet by Toshi https://itunes.apple.com/us/app/coinbase-wallet/id1278383455?mt=8

### PR DESCRIPTION
**Pull request by kennyt**
Closes #23

New google play link: https://play.google.com/store/apps/details?id=org.toshi
New apple link: https://itunes.apple.com/us/app/coinbase-wallet/id1278383455

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ import NavigationTabs from "../NavigationTabs";
```
Comment (chikeichan) [inline src/components/Header/index.js]: Can you update CipherLogo as well on line 7?

Here is the link to the logo: https://files.readme.io/b56e3b4-small-icon-512px.png

Please download it as png, and put in `assets/images/`

Comment (chikeichan) [inline src/components/Header/index.js]: Also, on line 61, can you update the copy to say `Coinbase Wallet` instead of `Cipher Browser`?

Comment (kennyt) [reply src/components/Header/index.js]: alright I replaced all instances of cipher in the codebase with coinbase wallet

Comment (kennyt) [reply src/components/Header/index.js]: done!

**Pull request merged**